Lifestyle/pomodoro.1s.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# <bitbar.title>Pomodoro timer</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Jan van den Berg</bitbar.author>
+# <bitbar.author.github>koozz</bitbar.author.github>
+# <bitbar.desc>Provides a minimal Pomodoro timer</bitbar.desc>
+# <bitbar.dependencies>bash</bitbar.dependencies>
+
+
+# Icon; http://emojipedia.org/tomato
+POMODORO="🍅"
+#POMODORO="\xF0\x9F\x8D\x85"  # echo -e $POMODORO
+POMODORO_SECONDS=1500
+POMODORO_DATA="$(dirname $0)/pomodoro.data"
```

Comment:
The $0 needs to be quoted otherwise if the path contains spaces then things could go wrong.